WizelineLabs/world-cup-predictions-2018
# Feat/custom leaderboard

### What does this PR do?
Adds the custom leaderboard

### Where should the reviewer start?
Leaderboard.vue

### How should this be manually tested?
- Sign In > Prediction Game > Leaderboard
- Try to search for a player
- A new table should appear when you add a new player
- Add other players
- Try to edit the table
- Remove a player
- Reset the custom table

### Any background context you want to provide?
One of the features more requested was the ability to create a custom leaderboard, so we created a way in which you can follow your friends.
